examples/filesystem.py

```diff
@@ -69,20 +75,19 @@ def _iter(self, path):
             size=st.st_size,
             uid=st.st_uid,
             gid=st.st_gid,
-            ctime=datetime.fromtimestamp(st.st_ctime),
-            mtime=datetime.fromtimestamp(st.st_mtime),
-            atime=datetime.fromtimestamp(st.st_atime),
+            ctime=datetime.fromtimestamp(st.st_ctime, tz=ZoneInfo("UTC")),
+            mtime=datetime.fromtimestamp(st.st_mtime, tz=ZoneInfo("UTC")),
+            atime=datetime.fromtimestamp(st.st_atime, tz=ZoneInfo("UTC")),
```

Comment:
we can just pass epoch values directly to the Record initializer now: 

```suggestion
            ctime=st.st_ctime,
            mtime=st.st_mtime,
            atime=st.st_atime,
```